storages/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.9.1'
+__version__ = '1.9.2'
```

Comment:
Please do not increase the version.

Reply:
Gotcha. My apologies. It's been removed.